src/api/java/CMakeLists.txt

```diff
@@ -9,8 +9,55 @@
 ## directory for licensing information.
 ##
 
-# create a directory for the cvc package
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/cvc")
+# create directories
+file(MAKE_DIRECTORY
+  "${CMAKE_CURRENT_BINARY_DIR}/src/main/resources"
+  "${CMAKE_CURRENT_BINARY_DIR}/src/main/java"
+  "${CMAKE_CURRENT_BINARY_DIR}/src/test/java"
+  )
+
+# find java
+find_package(Java)
```

Comment:
I would guess this is `REQUIRED` as well.

Reply:
It is implied by find_package(JNI REQUIRED). So far this line is redundant. I added it hoping it would fix the Mac build in github,  but it didn't.